File: MyNotesProject/CommonLibrary/src/main/java/com/fsk/common/presentation/utils/checkable_helper/OnCheckedChangeListener.java
package com.fsk.common.presentation.utils.checkable_helper;


import android.widget.Checkable;

/**
 * Interface definition for a callback to be invoked when the checked state of a compound button
 * changes.
 */
public interface OnCheckedChangeListener {

    /**
     * Called when the checked state changes.
     *
     * @param checkable
     *         The object whose state has changed.
     */
    void onCheckedChanged(Checkable checkable);
}

File: MyNotesProject/CommonLibrary/src/main/java/com/fsk/common/presentation/checkable/OnCheckedChangeListener.kt
package com.fsk.common.presentation.checkable

import android.widget.Checkable

/**
 * Interface definition for a callback to be invoked when the checked state of a compound button
 * changes.
 */
interface OnCheckedChangeListener {

    /**
     * Called when the checked state changes.

     * @param checkable
     * *         The object whose state has changed.
     */
    fun onCheckedChanged(checkable: Checkable)
}